source/patterns/@aws-solutions-constructs/aws-apigateway-dynamodb/test/integ.apigateway-dynamodb-CRUD.expected.json

```diff
@@ -653,5 +653,39 @@
         ]
       }
     }
+  },
```

Comment:
This seems like a rather arbitrary list of tests that were affected - what is common about these tests that led to the Bootstrap info now being included in the output?

Reply:
This is the new way of syn-thing the templates in CDK v2, which can be enabled in CDK v1 by setting [this](https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/cx-api/lib/features.ts#L35) feature flag. It will be default behavior in CDK v2 and there's no way to turn off this flag in CDK v2. Similar changes will apply to all the expected CFN templates